wandb/sdk/data_types/object_3d.py

```diff
@@ -32,9 +32,11 @@ class Object3D(BatchableMedia):
 
     The shape of the numpy array must be one of either:
     ```python
-    [[x y z],       ...] nx3
-    [x y z c],     ...] nx4 where c is a category with supported range [1, 14]
-    [x y z r g b], ...] nx4 where is rgb is color
+    [
+      [x y z],       ... nx3
+      [x y z c],     ... nx4 where c is a category with supported range [1, 14], each being a distinct color
+      [x y z r g b], ... nx6 where is rgb is color
+    ]
```

Comment:
```suggestion
    (X, Y, Z)  # 3d input
    (X, Y, Z, C) # 4d input where C is a category with supported range [1, 14], each being a distinct color
    (X, Y, Z, R, G, B) # 6d input where RGB is color
```

Reply:
The format of the data passed in to this datatype is a list of lists, which is why the n x 3, n x 4 and n x 6. Without the outer parenthesis and the comments saying 3d / 4d / 6d input, there is scope for misinterpretation

Reply:
i see but the text above says: "The shape of the numpy array must be one of either:" which implies one of the  bellow shapes... so i would suggest maybe adding another line that says that it could be a list of arrays of different shape?